main.tf

```diff
@@ -97,6 +97,7 @@ resource "aws_redshift_cluster" "this" {
 
   lifecycle {
     ignore_changes = [master_password]
+    prevent_destroy = var.prevent_deletion
```

Comment:
this is not valid HCL

Reply:
Hmm, I see. That's unfortunate. https://github.com/hashicorp/terraform/issues/22544